2021-offseason/src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import frc.robot.Constants;
+
+public class Intake extends SubsystemBase {
+
+  private TalonSRX intakeTalon;
+  private DoubleSolenoid solenoid;
+
+  public Intake() {
+    intakeTalon = new TalonSRX(0);
+    intakeTalon.setInverted(false);
```

Comment:
inversion state could/should also be a constant

Reply:
this still needs to be resolved @Nakul-Rajpal